Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN pip install setuptools Cython wheel numpy tensorflow-gpu pandas
 # install neuroglancer
 RUN mkdir /.ssh
 ADD ./ /neuroglancer
-RUN pip install -r /neuroglancer/python/requirements.txt
+RUN pip install -r /neuroglancer/python/requirements.txt --upgrade
```

Comment:
Why is upgrade required here?

Reply:
To be honest, I don't remember. I'll remove it.